core_backend/tests/api/test_question_dashboard.py

```diff
@@ -23,3 +52,7 @@ async def test_retrieve_question_dashboard(
     data = response.json()
     assert len(data["six_months_questions"]) == 6
     assert len(data["six_months_upvotes"]) == 6
+    assert all(value >= questions[0] for value in data["six_months_questions"])
```

Comment:
Instead of checking each values individually I make sure the values are within the treshold?

Reply:
My vote is to just make it fixed values. See comment above!